var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -93,5 +99,8 @@ def patch(self):
             "setup.py",
```

Comment:
This can be removed, it doesn't do anything

Reply:
Done